testsuite/src/test/groovy/noe/tomcat/BindingsTomcatConfiguratorIT.groovy

```diff
@@ -272,6 +272,34 @@ abstract class BindingsTomcatConfiguratorIT extends TomcatTestAbstract {
     assertEquals newValue, newProps.getProperty(propName)
   }
 
+  @Test
+  void addListenerSpecificListenerToConfigurator() {
```

Comment:
@jstefl Excuse me but i had not push the tests into this branch :P Here are the tests you  requested.I made also for addListener in order to be organised well